src/main/flight/rpm_filter.c

```diff
@@ -59,7 +59,7 @@ typedef struct rpmNotchFilter_s {
     float    maxHz;
     float    fadeRangeHz;
     float    q;
-    uint32_t looptime;
+    uint32_t looptimeUs;
```

Comment:
Good change. But you could change the `uint32_t` into a `timeUs_t` as well. :smiley: